src/mem/cache/prefetch/Prefetcher.py

```
@@ -737,6 +737,7 @@ class BOPPrefetcher(QueuedPrefetcher):
                               150, 160, 162, 180, 192, 200, 216, 225, 240, 243, 250, 256], "Predefined offsets")
 
     crossPage = Param.Bool(True, "Cross page prefetching")
+    enable_adaptoffset = Param.Bool(True, "enable adapt offset")
```

Ruff (0.14.7)

`Param` may be undefined, or defined from star imports

(F405)